packages/react-components/react-drawer/src/components/DrawerHeaderNavigation/useDrawerHeaderNavigation.ts

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { getNativeElementProps } from '@fluentui/react-utilities';
+import type { DrawerHeaderNavigationProps, DrawerHeaderNavigationState } from './DrawerHeaderNavigation.types';
+
+/**
+ * Create the state required to render DrawerHeaderNavigation.
+ *
+ * The returned state can be modified with hooks such as useDrawerHeaderNavigationStyles_unstable,
+ * before being passed to renderDrawerHeaderNavigation_unstable.
+ *
+ * @param props - props from this instance of DrawerHeaderNavigation
+ * @param ref - reference to root HTMLElement of DrawerHeaderNavigation
+ */
+export const useDrawerHeaderNavigation_unstable = (
+  props: DrawerHeaderNavigationProps,
+  ref: React.Ref<HTMLElement>,
+): DrawerHeaderNavigationState => {
+  return {
+    components: {
+      root: 'div',
```

Comment:
if it's a navigation element would `nav` be a better fit? might want to do the sacred a11y ritual and invoke @smhigley on this one :)

Reply:
Yeah. This should've been nav from the beginning. Fixed.